src/components/TimeTravel/_Timeline.js

```diff
@@ -75,19 +75,30 @@ const FullyPannableCanvas = styled.div`
 const TimelineContainer = FullyPannableCanvas.extend`
   background-color: ${props => transparentize(0.15, props.theme.colors.white)};
   box-shadow: inset 0 0 7px ${props => props.theme.colors.gray};
-  overflow: hidden;
+  /* overflow: hidden; */
```

Comment:
not sure if you meant to leave this here?

Reply:
Good catch, will remove it!